src/language/code_generator.py

```diff
@@ -287,9 +298,9 @@ def format_output(self, file):
             language: c++ or cmake
 
         """
-        if self.language == "c++" and self.app.clang_format:
-            LOGGER.debug("Formatting C++ file %s", str(file))
-            subprocess.check_call(self.app.clang_format + ["-i", str(file)])
+        if self.language == "c++" and not self.app.within_ci:
```

Comment:
A clarification question: if user doesn't have "compatible" clang format installed on his/her machine, what happens in this case? .i.e. if there is no clang-format installed or of a different version than we support, user will still be able to build the nmodl without any error?

Reply:
I think this is taken care of from within the coding convention tool that wraps clang-format?